sanic/app.py

```diff
@@ -481,9 +475,7 @@ def url_for(self, view_name: str, **kwargs):
 
         route = self.router.find_route_by_view_name(view_name, **kw)
         if not route:
-            raise URLBuildError(
-                f"Endpoint with name `{view_name}` was not found"
-            )
+            raise URLBuildError(f"Endpoint with name `{view_name}` was not found")
```

Comment:
Looks like my editor didn't read in the project's black/flake configuration properly. Will fix